riscv-page-tables/src/page_table.rs

```diff
@@ -624,7 +632,6 @@ impl<T: PagingMode> PageTableInner<T> {
 
 /// A paging hierarchy for a given addressing type.
 /// This is used for HS and HU paging modes such as Sv48.
-///
 /// TODO: Support non-4k page sizes.
```

Comment:
Can this TODO be removed now or there still work to be done to support non-4k pages?

Reply:
I actually asked for this to stay as there's quite a lot left -- we still need to support invalidating & unmapping hugepages, and then there's the whole interaction with `PageTracker`, which assumes everything is 4kB.

Reply:
Yes, just realised watching the history of reviewing.